index.json

```diff
@@ -134,8 +133,10 @@
   },
   "uniqueValues": {
     "firmware_version": [
+      "3.5",
```

Comment:
Configurator does support >= 4.0 - this would need configurator 10.8 not compatible with 4.4 firmware.